sonic-ycabled/tests/test_ycable.py

```diff
@@ -32,6 +33,20 @@
 
 os.environ["YCABLE_UNIT_TESTING"] = "1"
 
+exception_output_expected= """\
```

Comment:
will this work if python version changes in future?

Reply:
this would not be backwards compatible, thats why I had added some string checks only, 
Better to omit this check I think.